students/julienjass/Lesson02/print_grid.py

```diff
@@ -0,0 +1,50 @@
+#Part 1
+
+def makingGrid():
```

Comment:
per pep8 guidelines, should be snake_case -> `making_grid`

Reply:
Yes, I did submit it on the course website as well.